### DISCUSSION TITLE
Dab scan not working in 20220917

### DISCUSSION THREAD

**Discussion by skycamel** [rern/rAudio#1044] [Bug Reports]
I started from scratch with image i20220904 and updated to 20220917 and then installed DAB add-on.
When I perform a DAB scan in library I don't get any channels in library, but above the Dabradio icon it displays the figure 4.457.
When I go and check in the root (/) I can see the folders of the different ensembles created there instead of
in the /srv/http/data/dabradio folder, which is also not created. When I run the command dab-scanner-rtlsdr -F dabradio.txt the
file is created with all the ensembles inside.
After that I imported a backup.gz with the already scanned dab channels (from i20220904) and I can play the channels.
On a side note, also the webradio folder is kind of "messed up", where I now have my webradios in a sub-folder and the logo's are not displayed because they are in /srv/http/data/webradio/img/webradio.
Cheers

**Comment by Omnilocus** [comment]
I also found the same problem
